### PR TITLE
Update index.php qeqfag gw fq a

### PR DESCRIPTION
**Pull request by provjpzz**
hklfhqil fqf ;oqf;o qfo qjgq hgqh giqu iqhfk qfnq fman fkjq fu qopu 
gqgq gq g
gwgqg

- qgqgq
- gwgjq
- t151w
- gg
- gggqgq

### CODE REVIEW

**Comment by provjpzz** [comment]
ok

File: index.php [hunk 1]
```diff
@@ -2,4 +2,5 @@
 <h2>index</h2>
 <h3>index</h3>
 <h4>index</h4>
-<h5>index</h5>
\ No newline at end of file
+<h5>index</h5>
+<p>index</p>
```
Comment (nnam2501) [inline index.php:6]: â

File: index.php [hunk 2]
```diff
@@ -2,4 +2,5 @@
 <h2>index</h2>
 <h3>index</h3>
 <h4>index</h4>
-<h5>index</h5>
\ No newline at end of file
+<h5>index</h5>
```
Comment (nnam2501) [inline index.php:5]: alo 123

**Review by nnam2501 — APPROVED** [review]
aloalo 123123

**Comment by provjpzz** [comment]
a

**Pull request merged**